tests/Test 1 - Perform Login.spec.js

```diff
@@ -0,0 +1,22 @@
+const {test, expect } = require('playwright/test')
```

Comment:
Filename in js project should not contain spaces and special symbols. I would name this `performLogin.spec.js`, but it's up to you. But please, use camel case!

Reply:
fixed

Reply:
Now it seems like camel case tries to look like a snake case. [Camel case](https://developer.mozilla.org/en-US/docs/Glossary/Camel_case) should not have `_`